go.mod

```diff
@@ -61,6 +61,7 @@ require (
 	github.com/pingcap/tidb v1.1.0-beta.0.20220819091852-41b9e26d1ef4
 	github.com/pingcap/tidb-tools v6.1.1-0.20220715000306-1d2f00da8c3e+incompatible
 	github.com/pingcap/tidb/parser v0.0.0-20220819091852-41b9e26d1ef4
+	github.com/pkg/errors v0.9.1
```

Comment:
Where do we use `github.com/pkg/errors`, why it changes from indirect dependency.